### ISSUE TITLE
0.4.10 failed to build with fcitx 4.2.8

### ISSUE THREAD

**Issue by cheese** [fcitx/fcitx-configtool#15]
During cmake, it checks the required minimal version of fcitx is 4.2.6.

```
-- Checking for module 'fcitx-gclient>=4.2.6'
--   Found fcitx-gclient, version 4.2.8
```

But the build finally failed with:

```
cd /builddir/build/BUILD/fcitx-configtool-0.4.10/build/gtk3 && /usr/bin/cmake3 -E cmake_link_script CMakeFiles/fcitx-config-gtk3.dir/link.txt --verbose=1
/usr/lib64/ccache/cc  -Wall -Wextra -Wno-sign-compare -Wno-unused-local-typedefs -Wno-unused-parameter -fvisibility=hidden -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches   -m64 -mtune=generic    -Wl,-z,relro  CMakeFiles/fcitx-config-gtk3.dir/sub_config_parser.c.o CMakeFiles/fcitx-config-gtk3.dir/sub_config_widget.c.o CMakeFiles/fcitx-config-gtk3.dir/config_widget.c.o CMakeFiles/fcitx-config-gtk3.dir/configdesc.c.o CMakeFiles/fcitx-config-gtk3.dir/keygrab.c.o CMakeFiles/fcitx-config-gtk3.dir/main.c.o CMakeFiles/fcitx-config-gtk3.dir/main_window.c.o CMakeFiles/fcitx-config-gtk3.dir/im_widget.c.o CMakeFiles/fcitx-config-gtk3.dir/ui_widget.c.o CMakeFiles/fcitx-config-gtk3.dir/gdm-languages.c.o CMakeFiles/fcitx-config-gtk3.dir/im_dialog.c.o CMakeFiles/fcitx-config-gtk3.dir/common.c.o CMakeFiles/fcitx-config-gtk3.dir/im_config_dialog.c.o CMakeFiles/fcitx-config-gtk3.dir/dummy_config.c.o CMakeFiles/fcitx-config-gtk3.dir/fcitx-config-gtk3-resources.c.o  -o fcitx-config-gtk3 -rdynamic -lgtk-3 -lgdk-3 -latk-1.0 -lgio-2.0 -lpangocairo-1.0 -lgdk_pixbuf-2.0 -lcairo-gobject -lpango-1.0 -lcairo -lgobject-2.0 -lglib-2.0 -lfcitx-utils -lfcitx-config -lfcitx-utils -lfcitx-core -lfcitx-config -lfcitx-utils -lfcitx-gclient -lfcitx-utils -lgio-2.0 -lgobject-2.0 -lglib-2.0 -lpangocairo-1.0 -lgdk_pixbuf-2.0 -lcairo-gobject -lpango-1.0 -lcairo -lfcitx-config -lfcitx-core -lfcitx-gclient 
CMakeFiles/fcitx-config-gtk3.dir/sub_config_widget.c.o: In function `run_plugin':
/builddir/build/BUILD/fcitx-configtool-0.4.10/gtk3/sub_config_widget.c:274: undefined reference to `FCITX_ARRAY_SIZE'
/builddir/build/BUILD/fcitx-configtool-0.4.10/gtk3/sub_config_widget.c:298: undefined reference to `FCITX_ARRAY_SIZE'
/builddir/build/BUILD/fcitx-configtool-0.4.10/gtk3/sub_config_widget.c:298: undefined reference to `FCITX_ARRAY_SIZE'
CMakeFiles/fcitx-config-gtk3.dir/sub_config_widget.c.o: In function `open_native_file':
/builddir/build/BUILD/fcitx-configtool-0.4.10/gtk3/sub_config_widget.c:197: undefined reference to `FCITX_ARRAY_SIZE'
/builddir/build/BUILD/fcitx-configtool-0.4.10/gtk3/sub_config_widget.c:221: undefined reference to `FCITX_ARRAY_SIZE'
CMakeFiles/fcitx-config-gtk3.dir/sub_config_widget.c.o:/builddir/build/BUILD/fcitx-configtool-0.4.10/gtk3/sub_config_widget.c:221: more undefined references to `FCITX_ARRAY_SIZE' follow
collect2: error: ld returned 1 exit status
```

The environment is CentOS 7 with epel.

**Comment by 329379172** [comment]
i have same problem too

**Comment by cheese** [comment]
The patch does not fix the issue when building with fcitx 4.2.8.

**Comment by wengxt** [comment]
Ok, seems the required macro is only added after 4.2.8. Will bump the requirement.
